docs/python/doxygenlib/cdWriterDocstring.py

```diff
@@ -52,29 +52,6 @@ class Writer:
     # we can combine property docstrings for getters/setters.
     propertyTable = {}
 
-    # Default constructor that assumes we're processing a single module
-    def __init__(self):
-        #
```

Comment:
python overloads don't work like C++:  you can't just define a function twice.  this `__init__` was being replaced by the one below it, so `Writer()` never actually worked.  I've fixed the use of this class in `convertDoxygen`